src/app/page.tsx

```diff
@@ -1,29 +1,28 @@
-import IconChange from '@/components/UI/Icons/IconChange'
-import Image from 'next/image'
+'use client'
+
+import Button from '@/components/UI/Button'
+import { useApi, useApiContext } from '@/utils'
+import { notify } from '@/utils/notifications'
+
+const Home = () => {
+  const api = useApiContext()
+  const condition = true
```

Comment:
What's the purpose of the condition?

Reply:
Well, I understood :) I meant which condition could be there, for example?

Reply:
If you successfully fetched something else